pandas/core/arrays/boolean.py

```diff
@@ -257,6 +257,8 @@ class BooleanArray(BaseMaskedArray):
 
     # The value used to fill '_data' to avoid upcasting
     _internal_fill_value = False
+    TRUE_VALUES = {"True", "TRUE", "true", "1", "1.0"}
+    FALSE_VALUES = {"False", "FALSE", "false", "0", "0.0"}
```

Comment:
Can you move this to the module level? (they don't need to be attributes of the class, I think)

Reply:
Jeff asked a few comments above to make them to class variables?

Reply:
Ah, sorry, I missed that (now I think Jeff main intention of the comment was to move them out of the method). In the end I don't care that much whether it are class constants or module constants, so whatever is preferred. 
But if you keep it as class variables, you need to make them private (`_..`), because there is no need to expose this to the user, I think?

Reply:
Yeah, private is a good point. 
Don't have a strong feeling about this either.